lib/mapper.js

```diff
@@ -21,6 +26,143 @@ exports.identify = function(identify){
  * @api private
  */
 
-exports.track = function(track){
-  return track.json();
+exports.track = function(track, settings){
+  var result = {
+    app_group_id: settings.appGroupId,
+    src: "segment"
+  };
+
+  var event = track.event();
+  // if the event is "Completed Order" from ECommerce API, then track as purchases instead of event
+  if (event && event.match(/completed ?order/i)) {
```

Comment:
You actually don't need to do the event name matching here but can implement a separate completedOrder method, as in the Google Analytics implementation here: https://github.com/segmentio/integration-google-analytics/blob/master/lib/mapper.js#L75